src/bosh_azure_cpi/lib/cloud/azure.rb

```diff
@@ -54,7 +54,9 @@ module AzureCloud; end
 require 'cloud/azure/blob_manager'
 require 'cloud/azure/disk_manager'
 require 'cloud/azure/stemcell_manager'
-require 'cloud/azure/table_manager'
+require 'cloud/azure/meta_store/meta_store'
+require 'cloud/azure/meta_store/stemcell_meta'
+require 'cloud/azure/meta_store/table_manager'
```

Comment:
These three classes need unit tests.

Reply:
the code path are all covered.
so don't think need add unit test for these.
and also another reason we do not need to add this time is that I will replace the table manager with the new implementation.
will add it later if the code path are not covered then.

and I think the unit test is not the more the better, we should track the code coverage and try to improve that index, add the unit test when necessary, add unit test for every class is a extra burden for us.

Reply:
yes, if meta_store is mocked, then we need to add the unit test then.
will take care of it when the meta_store is mocked.(then the coverage will decline, we should not forget about it then :) )

Reply:
I agree with Bin. For unit test, it is not good to only depend on other components to verify these classes. We should add cases for them, at least we should have a future plan.

Reply:
yes, the future plan is "will add it later if the code path are not covered then."